jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -171,16 +171,7 @@ public void buildDocker()
 
     } catch (IncompatibleBaseImageJavaVersionException ex) {
       throw new GradleException(
-          "The base image uses Java "
-              + ex.getBaseImageMajorJavaVersion()
-              + ", but project is using Java "
-              + ex.getProjectMajorJavaVersion()
-              + "; perhaps you should configure a Java "
-              + ex.getProjectMajorJavaVersion()
-              + "-compatible base image using the 'jib.from.image' parameter, or set "
-              + "targetCompatibility = "
-              + ex.getBaseImageMajorJavaVersion()
-              + " or below in your build configuration");
+          HelpfulSuggestions.forIncompatibleBaseImageJavaVesionForGradle(8, 11));
```

Comment:
Should this just take in the exception instead?

Reply:
I didn't like making this class depend on the exception class.

Reply:
Should this at least take the versions from the exception instead of the hard-coded 8 and 11? Also, should prob add `ex` as the cause.

Reply:
Gosh. Nice catch.